frontend/src/api/axiosInstance.js

```
@@ -20,14 +22,19 @@ export function r(path) {
 export function a(path) {
     return r(`pdf/${path}`);
 }
-// Stelle sicher, dass der CSRF-Token-Interceptor aktiv ist (auskommentiert im Original)
 axiosInstance.interceptors.request.use((config) => {
     const csrftoken = Cookies.get('csrftoken');
+    if (config.data instanceof FormData) {
+        // Let the browser automatically set 'Content-Type: multipart/form-data; boundary=…'
+        // Do NOT manually set Content-Type for FormData - the browser handles this correctly
+        delete config.headers['Content-Type'];
+        // Don't set it back! The browser will add the correct boundary automatically
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)